src/pyfmi/fmi.pyx

```diff
@@ -1213,15 +1214,19 @@ cdef class FMUModelBase(ModelBase):
         self._fmu_temp_dir = NULL
         self._fmu_log_name = NULL
 
+        # Used to adjust behaviour if FMU is unzipped
+        self._allow_unzipped_fmu = 1 if allow_unzipped_fmu else 0
+
         #Specify the general callback functions
         self.callbacks.malloc  = FMIL.malloc
         self.callbacks.calloc  = FMIL.calloc
         self.callbacks.realloc = FMIL.realloc
         self.callbacks.free    = FMIL.free
         self.callbacks.logger  = importlogger
         self.callbacks.context = <void*>self #Class loggger
-        
-        
+
+        display_path_deprecation(path)
```

Comment:
I'm a bit unsure about the gain of adding an extra method just to display the message. I'd rather see the message directly here instead of invoking an extra method (and also even though this message will be shown in a few places)

Reply:
I agree that the function does not bring much, but I rather see the use of a function instead of the deprecation code duplicated in 5 other places. When we decide later to remove the path argument (as well as the deprecation) I believe it is easier if the developer simply needs to look for the calls to this function. What do you think?

Reply:
Ok